automation/taskcluster/lib/module_definitions.py

```diff
@@ -15,7 +15,9 @@ def from_gradle():
     if exit_code is not 0:
         print("Gradle command returned error: {}".format(exit_code))
 
-    gradle_modules = json.loads(output)
+    modules_line = [line for line in output.split('\n') if line.startswith('modules: ')][0]
```

Comment:
* Nit: maybe use splitlines() instead of `split('\n')? 
* Nit: s,`modules_line`,`modules_lines` since it's a list?

Reply:
There is a `[0]` at the end of the line :)